db/migrate/20160418123558_create_addresses.rb

```diff
@@ -0,0 +1,15 @@
+class CreateAddresses < ActiveRecord::Migration
+  def change
+    create_table :addresses do |t|
+      t.float :latitude
+      t.float :longitude
+      t.string :street_1
+      t.string :street_2
+      t.string :city
+      t.string :state_code
+      t.string :zip_code
```

Comment:
Might be good to add not null constraints to the columns with presence validations (`latitude`, `longitude`, `state_code`, `street_1`).